lib/delayed/job_groups/backend/active_record/railtie.rb

```diff
@@ -0,0 +1,13 @@
+module Delayed
+  module JobGroups
```

Comment:
Can we name this class `Delayed::JobGroups::Railtie` to be consistent with the namespacing of the other classes in this project?